starkbank/brcodepayment/__brcodepayment.py

```diff
@@ -148,7 +148,7 @@ def page(cursor=None, after=None, before=None, tags=None, ids=None, status=None,
 
 def update(id, status=None, user=None):
     """# Update BrcodePayment entity
-    Update an BrcodePayment by passing id, if it hasn't been paid yet.
+    Update a BrcodePayment by passing id, if it hasn't been paid yet.
```

Comment:
Talvez mudar para:
`Update a BrcodePayment by passing its id, if it hasn't been paid yet.`